### DISCUSSION TITLE
Control panel, taskbar personalization, boot/login screens, change username

### DISCUSSION THREAD

**Discussion by Pixel556** [DustinBrett/daedalOS#136] [Ideas]
Add control panel and taskbar personalization. Also add bootscreen and login screen. Add a 'change name' button on the top right corner to change the name. This features would be good for v3. (Note: don't put Windows logo on boot. use the start button icon on boot.)

**Comment by DustinBrett** [comment]
Thanks for the suggestions. I am mostly against a boot screen as I want the user to be able to interact asap. Same for login screen as I want the user to be auto logged in basically. As for the name and ability to login, I do indeed plan one day to allow users to go from Public to their own profiles. Control panel and personalization indeed are something I have planned, for that I am working on a new type of Dialog that I can use anywhere to ask the user configuration things in a GUI way.

**Comment by Pixel556** [comment]
Also: When will v3 be released? & Will you add control panel, personalization, and boot? (just add the welcome screen in place of logon)

**Reply by DustinBrett** [reply]
v3 will be a long term thing. Maybe end of this year perhaps. It isn't even a plan atm. Currently I am still adding features all the time. I will indeed be adding more customization. The long term goal of this project is to have everything relevant to a real OS, from a usability POV. But that could take a long time, like 2030+
